### DISCUSSION TITLE
Upvote/downvote UI

### DISCUSSION THREAD

**Discussion by islamCodehood** [dailydotdev/daily#1363] [General]
Hi,
I'd like to suggest this UI enhancement for the upvote/downvote buttons. My suggestion is to remove the separating line (div) between them. The images below to chow how it would look like after removing such line which I found more elegant.
![image](https://github.com/user-attachments/assets/6dc0b3f6-7cf5-4295-8c31-db99f549d66c)

![Screenshot 2024-08-25 154556](https://github.com/user-attachments/assets/66481f0b-dfc4-441f-a979-f31b1328f0a5)

**Comment by idoshamun** [comment]
Hi @islamCodehood,

Thanks for the effort in creating this mockup. I'll forward it to our design team to consider
